CreditManagement/models.py

```diff
@@ -289,6 +313,26 @@ def finalise(self):
             self.delete()
             fixed_transaction.save()
 
+        return fixed_transaction
+
+    def save(self, *args, **kwargs):
+        # If no confirm moment is given, set it to the standard
+        if self.confirm_moment is None:
+            self.confirm_moment = self.order_moment + settings.MINIMUM_BALANCE
```

Comment:
What does `MINIMUM_BALANCE` do here?

Reply:
That is indeed the wrong setting. It should be TRANSACTION_PENDING_DURATION  Don't know what happened here, I'll fix it.